packages/hooks/use-aria/index.ts

```diff
@@ -0,0 +1,26 @@
+import { pick } from 'lodash-unified'
+import { buildProps } from '@element-plus/utils'
+
+export const ariaProps = buildProps({
+  /**
+   * @description native `aria-label` attribute
+   */
+  ariaLabel: String,
+  /**
+   * @description native `aria-orientation` attribute
+   */
+  ariaOrientation: {
+    type: String,
+    values: ['horizontal', 'vertical', 'undefined'],
```

Comment:
Why is `'undefined'` added here? 🤔

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-orientation#values

Reply:
I see. 👍